specification/metrics/purefb-metrics.md

```diff
@@ -0,0 +1,115 @@
+|                                                                                                                                                                                                                           |                                                        |                                                                    |                  |                                                                                                                         |            | `dimension`          | `usec_per_other_op`, `usec_per_read_bucket_op`, `usec_per_read_object_op`, `usec_per_write_bucket_op`              |
+| Available                                                                                                                                                                                                                 | purefb_buckets_s3_specific_performance_throughput_iops | FlashBlade buckets S3 specific throughput in operations per second | second           | Gauge                                                                                                                   | Double     | `name`               | (name)                                                                                                             |
+|                                                                                                                                                                                                                           |                                                        |                                                                    |                  |                                                                                                                         |            | `dimension`          | `others_per_sec`, `read_buckets_per_sec`, `read_objects_per_sec`, `write_buckets_per_sec`, `write_objects_per_sec` |
+| Available <br />[Note: object_count space dimension to be removed in v1.0.10 in favor of purefb_buckets_object_count released in v1.0.8](https://github.com/PureStorage-OpenConnect/pure-fb-openmetrics-exporter/pull/42) | purefb_buckets_space_bytes                             | FlashBlade buckets space in bytes                                  | byte             | Gauge                                                                                                                   | Double     | `name`               | (name)                                                                                                             |
```

Comment:
because this will be a breaking change semver rules should apply. To comply fully for a breaking change we should go to v2.0.0, but I think we can get away with a change to v1.1.0

Reply:
what if we changed it to `in a future release` ?

Reply:
yes - that would be fine